src/FileUploader/FileUploader.test.tsx

```diff
@@ -29,6 +30,25 @@ const multipleFiles = [
   }
 ] as FileWithData[];
 
+/**
```

Comment:
This code looks identical between FileUploader and ImageUploader. That should be ringing alarm bells in your head and you should be making a single common function they both can use.